lib/Handlers/setV2.ts

```diff
@@ -0,0 +1,34 @@
+import { NodeType } from '@voiceflow/general-types/build';
+import { Node } from '@voiceflow/general-types/build/nodes/setV2';
+
+import { HandlerFactory } from '@/lib/Handler';
+
+import CodeHandler from './code';
+
+export type SetV2Options = {
+  endpoint?: string | null;
+  safe?: boolean;
+};
+
+const SetV2Handler: HandlerFactory<Node, SetV2Options | void> = ({ endpoint, safe } = {}) => ({
+  canHandle: (node) => node.type === NodeType.SET_V2,
+  handle: async (node, runtime, variables, program) => {
+    const codeHandler = CodeHandler({ endpoint, safe });
+
+    let code = `
+        let evaluated;
+    `;
+    node.sets.forEach((set) => {
```

Comment:
we should probably check that `set.variable` exists in `variables` - otherwise this is a guaranteed crashed

Reply:
how did they select the variable if it doesn't exist? easy check to make tho